pkg/ccl/backupccl/backup_planning.go

```diff
@@ -614,22 +608,22 @@ func backupPlanHook(
 		return nil, nil, nil, false, err
 	}
 
-	encryptionParams := backupEncryptionParams{}
+	encryptionParams := jobspb.BackupEncryptionOptions{Mode: jobspb.EncryptionMode_None}
 
 	var pwFn func() (string, error)
-	encryptionParams.encryptMode = noEncryption
+	encryptionParams.Mode = jobspb.EncryptionMode_KMS
```

Comment:
I don't think we should set this to KMS here, since the check inside `if backupStmt.Options.EncryptionKMSURI != nil` will error out if it sees a non-None value?

Reply:
Huh, whoops, yeah, this line is wrong and also pointless since I init the field above instead anyway. Good catch.